python/helpers/jailbreak_utils.py

```
@@ -0,0 +1,633 @@
+                result = subprocess.run([tool, "--help"],
+                                      capture_output=True,
+                                      text=True,
+                                      timeout=5)
+                tools_status[tool] = "✅ Available"
+            except (subprocess.TimeoutExpired, FileNotFoundError):
+                tools_status[tool] = "❌ Not found"
+
+        # Check for platform-specific tools
+        if self.system == "darwin":  # macOS
+            try:
+                result = subprocess.run(["brew", "--version"],
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)